packages/runtime/test-runtime-utils/src/insecureUsers.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IUser } from "@fluidframework/protocol-definitions";
+
+/**
+ * Insecure user definition.
+ * It extends the base IUser interface with a `name` property.
```

Comment:
Nit: Since this part of the description is non-semantic, I recommend moving it into a `remarks` block.
```suggestion
 * @remarks It extends the base IUser interface with a `name` property.
```